www/postfixadmin/Portfile

```diff
@@ -16,14 +16,13 @@ homepage            http://postfixadmin.sourceforge.net/
 master_sites        sourceforge:project/${name}/${name}/${name}-${version}
 
 platforms           darwin freebsd
-depends_run         port:php71 \
-                    port:mysql57 \
-                    port:apache2 \
+depends_run         port:apache2 \
+                    port:php73 \
```

Comment:
Since you are switching from php71 to php73, your commit message should mention this change.